acl/acl.go

```diff
@@ -266,6 +290,37 @@ func (p *PolicyACL) ServiceWrite(name string) bool {
 	return p.parent.ServiceWrite(name)
 }
 
+// EventRead is used to determine if the policy allows for a
+// specific user event to be read.
+func (p *PolicyACL) EventRead(name string) bool {
+	// Longest-prefix match on event names
+	if _, rule, ok := p.eventRules.LongestPrefix(name); ok {
+		switch rule {
+		case EventPolicyRead:
+			return true
+		case EventPolicyWrite:
+			return true
+		default:
+			return false
```

Comment:
Hitting this "default" covers the `deny` policy case for explicitly denial. The policy is checked during compilation to ensure that one of `read`, `write`, or `deny` is specified, so technically it could be `case EventPolicyDeny`, but it shouldn't be any different and I just followed the convention from the rest of the file.